app/src/main/res/layout/habit_row.xml

```diff
@@ -7,17 +7,56 @@
     android:layout_margin="5dp"
     android:id="@+id/habit_rows">
 
+    <LinearLayout
+        android:id="@+id/swipe_options"
+        android:layout_width="wrap_content"
+        android:layout_height="70dp"
```

Comment:
Constant height used here to ensure foreground and background views have same height to make swipe menu look clean.